lib/omni_exchange/providers/xe.rb

```diff
@@ -4,7 +4,7 @@
 
 module OmniExchange
   class Xe < Provider
-    ENDPOINT_URL = 'https://xecdapi.xe.com/'
+    ENDPOINT_URL = 'https://xecd-api-ssl.xe.com/'
```

Comment:
This is the actual change that I wanted to add ...this simply replaces the current XE's url with an alternative one that they provided while we sort things related with their SSL certificate. We are confirming if an actual change is needed or if ruby's open ssl with deal with it internally as long as it is a trusted certificate authority.